### ISSUE TITLE
iOS - Update Google dependencies?

### ISSUE THREAD

**Issue by cristianoccazinsp** [viromedia/virocore#320]
- [X] Review the documentation: https://virocore.viromedia.com/
  - [X] Search for existing issues: https://github.com/viromedia/virocore/issues
  - [X] Use the latest ViroCore release: https://virocore.viromedia.com/docs/releases
  - [X] Turn off Instant Run in Android Studio

## Environment
Please provide the following information about your environment:
1. OS: Mac
2. Version: ViroCore version - The one used by react-viro 2.17
3. Device(s): iPhone

## Description
Hello, opening this issue as a follow up from https://github.com/viromedia/viro/issues/718

Apple is rejecting apps that use UIWebView, and right now it seems like react-viro, which uses viro core, is referencing UIWebView somewhere.
Someone hinted that the references come from google's SDK, can those be updated? Can we get some help to remove any UIWebView reference?

Thanks.

## Reproducible Demo
-

**Comment by mstrokin** [comment]
@dthian If it's not possible to update the google SDK, is it possible to 'deprecate' the ViroViewGVR and other dependencies, thus dropping the support for stereo VR headsets, but still keeping the AR intact?

**Comment by Hless** [comment]
From what I understand we should replace the GVRSDK library with the new open source Cardboard library, which no longer contains references to UIWebView. They also appear to have some new features/fixes on the roadmap for that new library

**Comment by nickarora** [comment]
Bumping this; this is critical to any app that plans to use this in production.  Submissions to the AppStore will soon all be rejected because of the reference to UIWebView.  @dthian is there a plan to update the library?

**Comment by dthian** [comment]
@nickarora ViroMedia has no active roadmap or plans at the moment. However, as ViroCore is now open sourced (the build instructions are laid out within the ReadMe), please feel free to make the necessary changes if need be. 

If i've time, I might take a peak into this issue this weekend, but no guarantees.

**Comment by TwR-Slice** [comment]
any news on this issue? would it be possible to remove GVR without loosing AR supprt?

**Comment by brascene** [comment]
@nickarora exactly, we're currently in phase of submiting the app, and the timing couldn't be worse, and also our hands are tied to make any changes.. any ideas how could we replace GVRSDK?

**Comment by Jackyaung** [comment]
I am facing the same problem, any solution ？

**Comment by ktemby** [comment]
I was able to get through the UIWebView issue in my RN 0.62.2 project last night - details here https://github.com/viromedia/viro/issues/852#issuecomment-641810778
